code-challenges.test.js

```diff
@@ -50,3 +160,20 @@ const hand4 = [7, 2, 7, 2, 7]
 // Expected output: true
 
 // b) Create the function that makes the test pass.
+
+// Create a function fullHouse that take in an array
+const fullHouse = (array) => {
+
+
+// Create for loop to iterate through the array
+for(let i=0; i < array.length; ) 
+
+// Create a conditional statement 
+// if the array has 3 duplicate values && 2 Duplicate values return true
```

Comment:
Your logic in your pseudocode looks great, I would really like to see an attempt at the logic you wrote down.